README.md

```diff
@@ -77,7 +77,7 @@ The release file can be found inside `./core/build/distributions/`.
 ### Cleaning the build ###
     ./gradlew clean
 
-### Running a task with a particular version of Scala (either 2.11.x or 2.12.x) ###
+### Running a task with a particular *available* version of Scala (either 2.11.x, 2.12.x or 2.13.x) ###
```

Comment:
Nit: I'd edit it to be:

"Running a task with one of the Scala versions available (2.11.x, 2.12.x or 2.13.x)"

Reply:
Done.